### PR TITLE
test: write test for date

### PR DESCRIPTION
**Pull request by Sir-Nath**

### CODE REVIEW

**Comment by Yczar** [comment]
@developerjamiu can you check this out

File: test/core/text/word_test.dart [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import 'package:chance_dart/chance_dart.dart';
+import 'package:chance_dart/core/text/constants/words.dart';
+import 'package:test/test.dart';
+
+void main() {
+  test('verify that word returned is from a list of words',
+      () => expect(word(), isIn(words)));
```
Comment (Yczar) [inline test/core/text/word_test.dart:7]: can you add commas in between brackets

File: test/core/thing/src/animal_test.dart [hunk 2]
```diff
@@ -0,0 +1,8 @@
+import 'package:chance_dart/core/thing/constants/animals.dart';
+import 'package:chance_dart/core/thing/src/animal.dart';
+import 'package:test/test.dart';
+
+void main() {
+  test('verify that animal returned is random from animal list',
+      (() => expect(animal(), isIn(animals))));
```
Comment (Yczar) [inline test/core/thing/src/animal_test.dart:7]: can you add commas in between trailing brackets here

File: test/core/time/src/date_test.dart [hunk 3]
```diff
@@ -0,0 +1,9 @@
+import 'package:chance_dart/core/time/time.dart';
+import 'package:test/test.dart';
+
+void main() {
+  test(
+      'verify that year returned is the current date',
+      //18-11-2022 is the date as at when this test was done
+      (() => expect(date(), '18-11-2022')));
```
Comment (Yczar) [inline test/core/time/src/date_test.dart:8]: can you add commas in between trailing brackets here

**Review by Yczar — CHANGES_REQUESTED** [review]

**Review by Yczar — APPROVED** [review]

File: test/core/text/word_test.dart (hunk 1 above)
Comment (Sir-Nath) [reply test/core/text/word_test.dart]: done

File: test/core/thing/src/animal_test.dart (hunk 2 above)
Comment (Sir-Nath) [reply test/core/thing/src/animal_test.dart]: done

**Pull request merged**